packages/storybook/stories/CustomIcons/utils.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { mergeStyles, Stack, TextField } from '@fluentui/react';
+import React, { useState } from 'react';
+
+const iconGrid = (allIcons: Record<string, JSX.Element>, searchText?: string): JSX.Element[] => {
+  const allIconKeys = Object.keys(allIcons);
+  const filteredKeys = searchText
+    ? allIconKeys.filter((k) => k.toLowerCase().includes(searchText.toLowerCase()))
+    : allIconKeys;
+
+  return filteredKeys.map((key, idx) => {
```

Comment:
Can you also apply a sort() here? Looks like there's a chunk that aren't in order:
![image](https://user-images.githubusercontent.com/2684369/143656706-124dc120-b80e-44a2-9673-6fb8af7b933d.png)

Reply:
Fixed